windows-driver-docs-pr/develop/validating-windows-drivers.md

```diff
@@ -19,7 +19,65 @@ Use InfVerif with `/w` and `/v` to verify that a Windows Driver:
 * Meets the **declarative (D)** principle of [DCH Design Principles](dch-principles-best-practices.md)
 * Complies with the [driver package isolation](driver-isolation.md) requirement of [Getting Started with Windows Drivers](getting-started-with-windows-drivers.md)
 
-For more details, see [Running InfVerif from the command line](../devtest/running-infverif-from-the-command-line.md) for 
+For more details, see [Running InfVerif from the command line](../devtest/running-infverif-from-the-command-line.md).
+
+## Driver Verifier Driver Isolation Checks
```

Comment:
This text near the top of the page needs updating to account for this new validation method:
Use the InfVerif and ApiValidator tools to test your Windows Driver for compliance with the requirements described in [Getting Started with Windows Drivers](getting-started-with-windows-drivers.md).